src/sidebar/services/frame-sync.ts

```diff
@@ -59,7 +60,7 @@ export function formatAnnot({
 /**
  * Return the frame which best matches an annotation.
  */
-function frameForAnnotation(frames: Frame[], ann: Annotation) {
+function frameForAnnotation(frames: Frame[], ann: Annotation): Frame | null {
```

Comment:
Previously the return type was inferred as `Frame`. This is incorrect if the `frames` array is empty. That is not expected to happen in practice, but this service can't enforce that and we have test cases that check for this. I have changed this to be more explicit that there may be no matching frame.